email/1.3/modules/ROOT/edits/email-send.adoc

```diff
@@ -0,0 +1,363 @@
+:page-aliases: connectors::email/email-send.adoc
+
+Anypoint Connector for Email (Email Connector) can send messages over SMTP and SMTPS servers by using the *Send* operation. The following examples show how to configure Email Connector to send emails and attachments, in both Anypoint Studio and the XML editor:
+
+* <<send-email,Send Emails>> +
+Configure the Email Connector *Send* operation to send emails over an SMTPS server.
+
+* <<send-attachments,Send an Attachment>> +
+Configure the File Connector *Read* operation to read a JSON file and the Email Connector *Send* operation to send the file as an attachment over an SMTP server.
+
+* <<send-attachments,Send Multiple Attachments>> +
+Configure *Set Variable* components to define multiple media type attachments, and send these attachments using the Email Connector *Send* operation. Then log the result message using a *Logger* component.
+
+[[send-emails]]
+== Send Emails
+
+The following example illustrates how to send emails over the SMTPS server. The flow initiates with a *Scheduler* component, and then the *Send* operation sends the email.
+The SMTPS connection type enables SSL or TLS encryption and sends encrypted messages over the secured version of the SMTP server.
+
+The following screenshot shows the app flow for this example:
+
+.Email Connector Send messages over SMTPS
+image::email-send-flow.png[Email Connector Send messages over SMTPS flow]
+
+To create the flow: +
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, select the *Scheduler* component and drag it onto the canvas. +
+This source initiates a flow when a time-based condition is met.
+. Drag the Email *Send* operation to the right of the *Scheduler* source. +
+. On the *Send* configuration screen, click the plus sign (*+*) next to the *Connector configuration* field to configure a global element for the operation.
+. In the *Connection* field, select *SMTPS Connection*.
+. In the *General* tab, enter the following values:
++
+* *Host* +
+`pop.gmail.com`
+* *Port* +
+`995`
+* *User* +
+`user3@domain.com`
+* *Password* +
+`passwordconnection`
++
+[start=7]
+. In the *TLS Configuration* section, select *Edit inline*.
+. In the *Trust Store Configuration* section, enter the following values:
++
+* *Path* +
+`aTruststore.jks`
+* *Password* +
+`passwordtrust`
++
+[start=9]
+. In the *Key Store Configuration* section, enter the following values:
++
+* *Path* +
+`aKeystore`
+* *Password*
+`passwordkey`
++
+[start=10]
+. In the *Advanced* section, set *Enabled Protocols* to `TLSv1.2,SSLv3`.
+. Click *OK* to close the configuration window.
+
+The following screenshot shows the *Email SMTPS* configuration:
+
+.Email SMTPS configuration
+image::email-smtps-configuration.png[Email SMTPS configuration]
+
+[start=12]
+. In the *Send* operation configuration screen, set the following values:
++
+* *From address* +
+`user1@domain.com`
+* *To addresses* +
+`Edit inline`
```

Comment:
I'm still thinking about this formatting, and I am asking for input....

The style ref. recommends inserting a blank line space between a list item and its description, which is perfectly fine in most cases and also helps with scanning and thinking about the elements as list item + separate paragraph.
But in lists like this one, that recommendation results in a lot of space:

```suggestion
. In the *Send* operation configuration screen, set the following values:
+
* *From address* +
+
`user1@domain.com`
* *To addresses* +
+
`Edit inline`
```
I hate to break my own rules, but when the description IS THIS SHORT, I think we might actually want to use this format (which I would never recommend in any other situation):

```suggestion
. In the *Send* operation configuration screen, set the following values:
+
* *From address*: `user1@domain.com`
* *To addresses*: `Edit inline`
```
What do YOU think?

Reply:
Sorry! Back to COPYEDITING ONLY....